src/code/z_kaleido_scope_call.c

```diff
@@ -1,56 +1,60 @@
 #include "global.h"
 #include "vt.h"
 
-void (*sKaleidoScopeUpdateFunc)(GlobalContext*);
-void (*sKaleidoScopeDrawFunc)(GlobalContext*);
-f32 D_80161398;
+void (*sKaleidoScopeUpdateFunc)(GlobalContext* globalCtx);
+void (*sKaleidoScopeDrawFunc)(GlobalContext* globalCtx);
+f32 gBossMarkScale;
 u32 D_8016139C;
-void* D_801613A0;
+PauseMapMarksData* gLoadedPauseMarkDataTable;
 
-extern void KaleidoScope_Update(GlobalContext*);
-extern void KaleidoScope_Draw(GlobalContext*);
+extern void KaleidoScope_Update(GlobalContext* globalCtx);
+extern void KaleidoScope_Draw(GlobalContext* globalCtx);
 
 void KaleidoScopeCall_LoadPlayer() {
-    KaleidoManagerOvl* playerActorOvl = &gKaleidoMgrOverlayTable[KALEIDO_OVL_PLAYER_ACTOR];
+    KaleidoMgrOverlay* playerActorOvl = &gKaleidoMgrOverlayTable[KALEIDO_OVL_PLAYER_ACTOR];
 
     if (gKaleidoMgrCurOvl != playerActorOvl) {
-        if (gKaleidoMgrCurOvl) {
+        if (gKaleidoMgrCurOvl != NULL) {
             osSyncPrintf(VT_FGCOL(GREEN));
-            osSyncPrintf("カレイド領域 強制排除\n"); // Kaleido area forced exclusion
+            osSyncPrintf("カレイド領域 強制排除\n"); // "Kaleido area forced exclusion"
             osSyncPrintf(VT_RST);
+
             KaleidoManager_ClearOvl(gKaleidoMgrCurOvl);
         }
+
         osSyncPrintf(VT_FGCOL(GREEN));
-        osSyncPrintf("プレイヤーアクター搬入\n"); // Player actor import
+        osSyncPrintf("プレイヤーアクター搬入\n"); // "Player actor import"
         osSyncPrintf(VT_RST);
+
         KaleidoManager_LoadOvl(playerActorOvl);
     }
 }
 
 void KaleidoScopeCall_Init(GlobalContext* globalCtx) {
-    // Kaleidoscope replacement construct
-    osSyncPrintf("カレイド・スコープ入れ替え コンストラクト \n");
+    osSyncPrintf("カレイド・スコープ入れ替え コンストラクト \n"); // "Kaleidoscope replacement construction"
+
     sKaleidoScopeUpdateFunc = KaleidoManager_GetRamAddr(KaleidoScope_Update);
     sKaleidoScopeDrawFunc = KaleidoManager_GetRamAddr(KaleidoScope_Draw);
 
     LOG_ADDRESS("kaleido_scope_move", KaleidoScope_Update, "../z_kaleido_scope_call.c", 98);
     LOG_ADDRESS("kaleido_scope_move_func", sKaleidoScopeUpdateFunc, "../z_kaleido_scope_call.c", 99);
     LOG_ADDRESS("kaleido_scope_draw", KaleidoScope_Draw, "../z_kaleido_scope_call.c", 100);
     LOG_ADDRESS("kaleido_scope_draw_func", sKaleidoScopeDrawFunc, "../z_kaleido_scope_call.c", 101);
+
     KaleidoSetup_Init(globalCtx);
 }
 
 void KaleidoScopeCall_Destroy(GlobalContext* globalCtx) {
-    // Kaleidoscope replacement destruction
-    osSyncPrintf("カレイド・スコープ入れ替え デストラクト \n");
+    osSyncPrintf("カレイド・スコープ入れ替え デストラクト \n"); // "Kaleidoscope replacement destruction"
+
```

Comment:
I feel like the comment should either be on top or under the call.

Reply:
on the side is fine, we do it in plenty of places. would not do under